docs/ci/jobs.md

```diff
@@ -89,3 +89,27 @@ build? This is most likely it.**
 
 UI tests for Core. See artifacts for a comprehensive report of UI. See [tests/ui-tests](../tests/ui-tests.html#reports)
 for more info.
+
+### hardware core regular device test
+
+[Device tests](../tests/device-tests.md) that run against an actual physical Trezor T.
+The device needs to have special bootloader, found in `core/embed/bootloader_ci`, that
+makes it possible to flash firmware without confirmation on the touchscreen.
+
+All hardware tests are run nightly on the `master` branch, as well as on push to branches
+with whitelisted prefix. If you want hardware tests ran on your branch, make sure its
+name starts with `hardware/`.
```

Comment:
It's `hw/` not `hardware/` isn't it? https://github.com/trezor/trezor-firmware/blob/master/ci/test-hw.yml#L12

Reply:
D'oh, not like didn't write hw/mmilata/tt-tests a hundred times lately ... fixed in 4172d48.